README.md

```diff
@@ -1,87 +1,55 @@
+# HPCC Systems Platform Community Version
+  platform/
+  -  ca/base/<version>/Dockerfile: HPCC Systems Platform Community version prequisites (Ubuntu 18.04)
```

Comment:
the -version- <version> tag ONLY displays in this raw mode, it does not display in rich text display... use &lt; (& l t ; and & g t ; ) and &gt; or perhaps change <version> to _version_